modules/nf-core/cutadapt/meta.yml

```diff
@@ -0,0 +1,49 @@
+name: cutadapt
```

Comment:
This looks module code that is installed. This does not need to be saved in the git repository. Users can install this themselves with a nextflow command I presume?